### ISSUE TITLE
Server not support diffie-hellman-group1-sha1

### ISSUE THREAD

**Issue by legionpheonix** [Terminals-Origin/Terminals#83]
When connecting via SSH to pfssense, it produces such an error:
server does not support diffie-hellman-group1-sha1 for keyexchange

Through putty it goes without any problems.

You can add the option not to use diffie-hellman-group1-sha1 when connecting?

**Comment by jirkapok** [comment]
No, see #46 , where we are going to replace current implementation by putty integration. If putty options satisfy your needs, you should be fine.

**Comment by legionpheonix** [comment]
Then I really look forward to a new update. Thank you so much.
Putty is the very option.

**Comment by Lillecarl** [comment]
@jirkapok When's a new release planned?
